examples/Bunch/include/simulation_defines/param/fieldBackground.param

```diff
@@ -44,11 +44,11 @@ namespace picongpu
 
         /* TWTS E-fields need to be initialized on host,
          *  so they can look up global grid dimensions. */
-        PMACC_ALIGN(twtsFieldE, const templates::twts::EField);
-        
+        const templates::twts::EField twtsFieldE;
+
         /* Constructor is host-only, because of subGrid and halfSimSize initialization */
         HINLINE FieldBackgroundE( const float3_64 unitField ) : unitField(unitField),
```

Comment:
If possible please move `unitField(unitField)` to a new line thus all initializations has the same indention.